app/controllers/items_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ItemsController < ApplicationController
+
+  def new
+    @user = current_user
+    @item = Item.new 
+#    authorize @item
+  end
+
+  def create
+    @user = current_user
+    @items = @user.items
+
+    @item = current_user.items.build( items_params )
+    @new_item = Item.new
+    
+#    authorize @item
+    if @item.save
+      flash[:notice] = "New to-do item created."
+    else
+      flash[:error] = "There was an error creating your to-do. Please try again."
```

Comment:
I added validation to ensure that items can't be saved if they are blank. I tested this and can see that the item is not being saved but (as above) no error flashes.